src/app/components/login-page/login-page.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { LoginService } from '../../services/login-service/login-service.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss']
+})
+export class LoginPageComponent implements OnInit {
+
+  usernameInput: string;
+  passwordInput: string;
+  loginMessage: string;
+
+  constructor(private loginService: LoginService, public router: Router) { }
+
+  sendLogin() {
+    if (this.usernameInput && this.passwordInput) {
+      this.loginService.login(this.usernameInput, this.passwordInput).subscribe((response) => {
+        if (response.status === 200) {
+          this.router.navigate([`/heroes-hall/${response.body.id}`]);
+        }
+      },
+      error => {
+        this.loginMessage = error.error.message;
```

Comment:
The serverside error message might not match the frontend error message. If we want to support multiple languages, you want to receive and error code or a standardized error message from the server, and show the proper error message here.
Write some logic to show messages.

Reply:
Added logic to deal with this that can be expanded in the future as well.